Sources/AccountSDKIOSWeb/Resources/da.lproj/Localizable.strings

```diff
@@ -1,8 +1,11 @@
-"SimplifiedWidget.continueToLogIn" = "Fortsæt med at logge ind";
+"SimplifiedWidget.logInWithOneClick" = "Log ind med et enkelt klik";
+"SimplifiedWidget.loginIncentive" = "Du er allerede logget ind på en af
+Schibsteds ydelser.";
```

Comment:
```suggestion
"SimplifiedWidget.loginIncentive" = "Du er allerede logget ind på en af Schibsteds ydelser.";
```

Reply:
Oh yeah, I was mostly concerned about the extra newline, which seemed like a typo. It's better to use `\n` in strings files than letting it span multiple lines.

Reply:
Good point!